utils/import.go

```diff
@@ -7,12 +7,11 @@ import (
 	"os"
 	"strings"
 
-	"github.com/maaslalani/invoice/utils"
 	"github.com/spf13/pflag"
 	"gopkg.in/yaml.v3"
 )
 
-func importData(path string, structure *utils.Invoice, flags *pflag.FlagSet) error {
+func ImportData(path string, structure *Invoice, flags *pflag.FlagSet) error {
```

Comment:
is switching away from camelCase necessary?

Reply:
Go modules export by case. Uppercase functions get exported, lowercase are private to the module